.env.template

```diff
@@ -11,4 +11,5 @@ OPENAI_API_BASE=your-base-url-for-azure
 OPENAI_API_VERSION=api-version-for-azure
 OPENAI_DEPLOYMENT_ID=deployment-id-for-azure
 IMAGE_PROVIDER=dalle
-HUGGINGFACE_API_TOKEN=
\ No newline at end of file
+HUGGINGFACE_API_TOKEN=
+USE_MAC_OS_TTS=False
```

Comment:
I think we don't need an empty line here

Reply:
it just looks like it, there is no line between 14 and 15

Reply:
LGTM then